vue3-apollo-quasar/src/components/PaginationButtons/PaginationButtons.vue

```diff
@@ -1,15 +1,15 @@
 <template>
   <div class="row justify-center items-center q-my-md">
     <button
```

Comment:
We had discussed previously that once we got to the pagination, we'd use the quasar component.

Got mulltiple concerns with this: 
- Using HTML `button` component despite the discussion to prefer quasar components
- Creating this component altogether since quasar already has[ a prebuilt pagination component](https://quasar.dev/vue-components/pagination) that would work much better than trying to reinvent the will

## TL;DR
Kindly use the [quasar pagination components](https://quasar.dev/vue-components/pagination) for this and customize styling if need-be.

Reply:
@hdJerry